src/assignment1/Program.cs

```diff
@@ -0,0 +1,47 @@
+﻿// <copyright file="Program.cs" company="PlaceholderCompany">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
+
+namespace Assignment1
+{
+    /// <summary>
+    /// program class holds the <see cref="Main(string[])"/> class.
+    /// </summary>
+    internal class Program
+    {
+        /// <summary>
+        /// Main class is used to do mathematical operations with the use of <see cref="MathMethod"/> method.
+        /// </summary>
+        /// <param name="args"> arguments </param>
+        public static void Main(string[] args)
+        {
+            MathMethod();
+            Console.ReadKey();
+        }
+
+        /// <summary>
+        /// MathMethod method is used get input and do mathematical operation with <see cref="Math"/> class.
+        /// </summary>
+        private static void MathMethod()
```

Comment:
Please update the method name with a meaningful name.
```suggestion
        private static void ArithmeticOperations()
```

Reply:
MathMethod method is renamed to "MathOperations".